.cypress/cypress/integration/peterborough.js

```diff
@@ -38,4 +38,15 @@ describe('new report form', function() {
     cy.get('.category_meta_message').should('contain', 'You can pick a tree from the map');
   });
 
+  it('displays nearby roadworks', function() {
+    cy.fixture('peterborough_roadworks.json');
+    cy.route('/streetmanager.php**', 'fixture:peterborough_roadworks.json').as('roadworks');
+    cy.wait('@roadworks');
+    cy.get('select:eq(4)').select('Pothole');
+    cy.contains('Roadworks are scheduled near this location');
+    cy.contains('Parapet improvement');
```

Comment:
These should probably be suffixed with `.should('be.visible')` after Matthew's recent comment.